http-api/src/lib.rs

```diff
@@ -479,6 +466,19 @@ fn get_project_metadata(urn: &Urn, paths: &Paths) -> Result<project::Metadata, E
     Ok(meta)
 }
 
+fn get_head_commit(urn: &Urn, paths: &Paths, default_branch: &str) -> Result<git::Commit, Error> {
+    let repo = git::Repository::new(paths.git_dir().to_owned())?;
+    let namespace = git::namespace::Namespace::try_from(urn.encode_id().as_str())?;
+    let browser = git::Browser::new_with_namespace(
+        &repo,
+        &namespace,
+        git::Branch::local(&default_branch),
+    )
+    .map_err(|_| Error::MissingLocalState)?;
+    let history = browser.get();
```

Comment:
```suggestion
    let history = browser.get();
    
```
Blank line after declarations